README.md

```diff
@@ -38,13 +38,15 @@ Encrypted vars can be added to `.travis.yml` using the travis cli tools as decri
 Currently configured with GITHUB_TOKEN value.
 
 ### To Production
-The production is hosted using (now.sh)[https://now.sh].
+The production is hosted using [now.sh](https://now.sh).
 The process is manual and requires deploying two now apps. You can do that by installing now and deploying the separate now.json files.
 To speed this up you can make sure now is installed globally before running npx `yarn global add now`.
+This will deploy what is currently in your dist folder, so make your have a build in `dist/`
 
 Quickstart:
 ```bash
 ./build.sh
+yarn build
```

Comment:
Should we incorporate the `build.sh` into the `yarn build`? Or rename one? 2 commands might be confusing.

Reply:
I think that we should rethink build.sh and make it so `yarn install` installs the dependency on contemplating action. That to me sounds like a re-arch job in ca.

Reply:
> so `yarn install` installs the dependency on contemplating action.

This sort of re-shape will make it so we can hook up auto deploys. Right now build.sh can't execute on the remote machine